### ISSUE TITLE
Patient photo is not saved

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#253]
```
Patient photo is not saved
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 19 Dec 2012 at 10:14

**Comment by GoogleCodeExporter** [comment]
```
not only that it's not saved.
The Patientbar and the IdentityViewer shows another file (the one from db) as 
the ThemedGenderButton (default icon).
```

Original comment by `christian.a.reiter@gmail.com` on 19 Dec 2012 at 10:31

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision be14918cd3fa.

The photo is now correctly saved/retrieved from patient database
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 53fd0aa9db0e.

Identity viewer photo under the patient search widget gets correctly updated
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision fb2e573e9c4f.
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 11:13
- Changed state: **Fixed**
